modules/meter/main/index.coffee

```diff
@@ -89,89 +91,90 @@ class Meter extends hx.EventEmitter
 
     size = Math.min(selection.width(), selection.height() * 2)
 
-    # 14 is the standard font size, 150 is the assumed size, and size is the actual
-    # size of the container. This scales the fonts appropriately
-    container.style('font-size', Math.round(14 / 150 * (size / 2)) + 'px')
-    textOffset = 10 * (size / 2) /  150
+    if size > 0
```

Comment:
If size is 0 do nothing and return the meter. Currently it doesn't emit the `render` event when the size is 0, is this the right behaviour?